.github/workflows/test.yml

```diff
@@ -10,84 +10,60 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    env:
-      redshift-version: redshift
-
     strategy:
       matrix:
         include:
-          - node-version: 10.x
-            mysql-version: 5.7
-            maraidb-version: 10.1
-            cassandra-version: 2.1
-            postgres-version: 9
-            sqlserver-collation: Latin1_General_CI_AS
```

Comment:
I would ask that we not remove the testing on `sqlserver-collation` as it's a useful check. This package was broken on case sensitive for a long while and beekeeper has run afoul of it a few times as well.

Reply:
We still setting that for tests. I just moved it directly to the docker compose file because it was the same value for all tests https://github.com/sqlectron/sqlectron-db-core/pull/28/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R67

Reply:
Just to clarify, it's `Latin1_General_CI_AS` for Node 10 and 14, and `Latin1_General_CS_AS` for Node 12 (`CI` vs `CS`) in the test suite here previously.

Just to be clear, I'm not objecting to setting it in the docker-compose as that's totally the right place for it, just that we do need that environment variable for it.

Reply:
ah I didn't notice there was letter different in one of them 🤦